### DISCUSSION TITLE
When can to support install via yarn?

### DISCUSSION THREAD

**Discussion by carveybunt** [tailwindlabs/tailwindcss#4819] [Ideas]
Is it possible?

**Comment by simonswiss** [comment]
Hey!

If you're asking if it's possible to do 

```sh
yarn add tailwindcss autoprefixer postcss
```

instead of

```sh
npm install tailwindcss autoprefixer postcss
```

The answer is **yes**, it's absolutely possible.

In most of our examples we use `npm`, but I personally use `yarn` instead everytime I'm _not on camera_ 😅

**Reply by carveybunt** [reply]
Thanks a lot, I got it. And I very like your video on youtube which helps me too more.

**Reply by jenstornell** [reply]
I was kind of wondering the same.

For us who don't want to struggle with setup, I think it would be great to add it in the docs:

https://tailwindcss.com/docs/installation
https://tailwindcss.com/docs/guides/vue-3-vite

...and make a video about it. Your videos are brilliant Simon!
